apps/vr-tests/package.json

```diff
@@ -22,6 +22,7 @@
     "@types/webpack-env": "1.13.0",
     "@uifabric/prettier-rules": ">=1.0.1 <2.0.0",
     "@uifabric/tslint-rules": ">=1.0.1 <2.0.0",
+    "@uifabric/date-time": "0.3.3",
```

Comment:
You should run `npm run rush-update` (and commit the result) to get this added to the shrinkwrap.

Reply:
I knew I was missing something, thanks!